SnapSheet/Classes/SnapSheetViewController.swift

```diff
@@ -10,6 +10,8 @@ import UIKit
 
 public class SnapSheetViewController: UIViewController {
 
+	public var didUpdateSheetOriginTop: ((CGFloat) -> (Void))?
```

Comment:
PP/NTH: didUpdateSheetFrame (might be interesting to have the full frame instead of just its top) + document in which coordinate system it is.

Alternatively: didUpdateSheetFrame: ((UIView) -> Void)? , so UIView's methods can be used to convert between coordinate systems